CHANGELOG

```diff
@@ -1,6 +1,19 @@
 CHANGELOG
 =========
 
+Version 0.11
+-----------
+
+- Drop support for Python 3.8 #64
+- Drop support for Django 3.2, 4.0, 4.1, 5.0 #64
```

Comment:
Although this would typically be a single PR, I split it into multiple lines for better readability.